lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -6824,28 +6829,18 @@ void EmitDestructuredObject(ParseNode *lhs,
 
     if (pnode1 != nullptr)
     {
-        Js::RegSlot propIdArrLocation = 0;
-        Js::RegSlot computedPropIdArrLocation = 0;
+        Js::RegSlot propIdArrLocation = Js::Constants::NoRegister;
+        Js::RegSlot computedPropIdArrLocation = Js::Constants::NoRegister;
         if (hasRest)
         {
             uint extraAlloc = 0, auxSize = 0;
-            if (UInt32Math::Mul(staticCount, sizeof(Js::PropertyId), &extraAlloc)
-                || UInt32Math::Add(sizeof(Js::PropertyIdArray), extraAlloc, &auxSize))
-            {
-                ::Math::DefaultOverflowPolicy();
```

Comment:
Why do you not need `DefaultOverflowPolicy` anymore?

Reply:
Akrosh said it's already checked in the arithmetic function

---
In reply to: [205494661](https://github.com/Microsoft/ChakraCore/pull/5498#discussion_r205494661) [](ancestors = 205494661)